distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -225,7 +225,7 @@ type Vector a
 
     ## Combines all the elements of the vector, by iteratively applying the
        passed function with the next element of the vector. After each step the
-       value is stored resulting in a new Vector of the same size as self.
+       value is stored resulting in a new vector of the same size as self.
```

Comment:
Whenever there is `vector` in lowercase, it is replaced by `array` when comparing the documentation in the `Array` type.